### ISSUE TITLE
Incompatible with PhpStorm 2019.2

### ISSUE THREAD

**Issue by majun** [PaleoCrafter/Chronomorph#4]
![image](https://user-images.githubusercontent.com/952656/63271683-f56fae80-c29a-11e9-8011-c547783304ab.png)

**Comment by quberok** [comment]
Would be great if someone will fix that.... This is very useful feature.

**Comment by softhints** [comment]
I was able to use the plugin with the following steps:

1. Update the project for the new version. This is the fork: https://github.com/softhints/Chronomorph
2. Clone the project in IntelliJ
3. Select your latitude and longitude - update file( this step is optional - but helped me to fix the problem with saving the latitude and longitude - so I predefined the daylight usage): 
https://github.com/PaleoCrafter/Chronomorph/blob/master/src/main/kotlin/io/paleocrafter/chronomorph/ChronomorphSettings.kt
4. Go to Gradle tasks (from top right corner) - then select intellij/build plugin
5. Open PyCharm / IntelliJ / PHPStorm - and install the plugin manually from this folder:
`~/Chronomorph/.sandbox/plugins/Chronomorph/lib`

Hope this help.
